src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/swift/SwiftAPIClient.java

```diff
@@ -2,194 +2,134 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Paths;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.Objects;
 import java.util.Properties;
-import java.util.concurrent.Executors;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.TimeUnit;
 
+import org.apache.commons.io.IOUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
 import org.apache.log4j.Logger;
-import org.fogbowcloud.saps.engine.core.util.ProcessUtil;
+import org.fogbowcloud.saps.engine.core.archiver.storage.exceptions.InvalidPropertyException;
+import org.fogbowcloud.saps.engine.core.dispatcher.email.keystone.IdentityToken;
+import org.fogbowcloud.saps.engine.core.dispatcher.email.keystone.KeystoneV3IdentityRequestHelper;
 import org.fogbowcloud.saps.engine.utils.SapsPropertiesConstants;
 import org.fogbowcloud.saps.engine.utils.SapsPropertiesUtil;
+import org.json.JSONException;
 
 public class SwiftAPIClient {
 
-    // Properties
-    private Properties properties;
+    private static final Logger LOGGER = Logger.getLogger(SwiftAPIClient.class);
+    private static final String CONTAINER_URL_PATTERN = "%s/%s?path=%s";
 
-    // Core Attributes
-    private String projectId;
-    private String userId;
-    private String userPassword;
-    private String tokenAuthUrl;
-    private String swiftUrl;
-    private String token;
+    private final String swiftUrl;
+    private final String tokenAuthUrl;
```

Comment:
``` private final String tokenAuthUrl;
    private final String projectId;
    private final String userId;
    private final String userPassword;``` do not need to be instance variables. they could stay at the local context in the constructor

Reply:
Done.